src/test/java/application/models/TestAnswerDTO.java

```diff
@@ -1,5 +1,6 @@
 package application.models;
 
+import application.models.dto.AnswerDTO;
```

Comment:
How about moving the DTO tests into similar structure?  test/java/application/models/dto/TestAnswerDTO.java?

Reply:
Good catch. Fixed.